README.md

```diff
@@ -1,2 +1,41 @@
 # Portfolio
 My personal portfolio
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> This is My personal portfolio project, I'm starting off with HTML and CSS and I plan to add more features to this repository in the future and eventually use it as my main portfolio!
+
+## Built With
+
+- Html & Css
+
+## Authors
+
+👤 **Emmanuel Cortés**
+
+- GitHub: [@emmanuelcortes2](https://github.com/emmanuelcortes2)
+- Twitter: [@EmmanueCortesP](https://twitter.com/EmmanuelCortesP)
+- LinkedIn: [Emmanuel Cortés Padilla](www.linkedin.com/in/emmanuel-cortés-padilla-490982140)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] Please make sure when the `issues page` link is clicked it takes you to the issues page of your project. You can do this by replacing `../../issues/` with your project issues page link.